input/postgres/relation_stats.go

```diff
@@ -156,3 +170,51 @@ func GetIndexStats(db *sql.DB, postgresVersion state.PostgresVersion, ignoreRege
 
 	return
 }
+
+func GetColumnStats(logger *util.Logger, db *sql.DB, globalCollectionOpts state.CollectionOpts, systemType string) (columnStats state.PostgresColumnStatsMap, err error) {
+	var sourceTable string
+
+	helperExists := false
+	db.QueryRow(QueryMarkerSQL + columnStatsHelperSQL).Scan(&helperExists)
+
+	if helperExists {
+		logger.PrintVerbose("Found pganalyze.get_column_stats() stats helper")
+		sourceTable = "pganalyze.get_column_stats()"
+	} else {
+		if systemType != "heroku" && !connectedAsSuperUser(db, systemType) && globalCollectionOpts.TestRun {
```

Comment:
Oof, so `pg_monitor` is not enough here, right? I just tested and that seems to be the case. We'll always need the helper to collect stats unless running as superuser...

This is unfortunate, since stats for columns we don't have permissions to collect are just silently excluded. This means that anyone upgrading the collector will also need to run this additional step--it's outside the scope of this patch, but we should think about that workflow or we just won't get anyone actually collecting any column stats (I expect putting something in the release notes is not enough).

Reply:
How did you test this, exactly? I had just copied this check from `input/postgres/relation_bloat.go` because it also uses `pg_stats`. Though it's slightly different from that version because of @lfittl's comment earlier: https://github.com/pganalyze/collector/pull/192#discussion_r689674107

Reply:
Ah, I missed Lukas' comment--I just tested locally: I created a new role in my local Postgres and checked if it could see pg_stats rows for a table from my default user (it could not). I granted `pg_monitor` to the new role and it could still not see those stats. I then granted my user to that role and it could see them. I think the check is right, and there's nothing else to do in the PR--we just need to think about how to communicate that users need to add the helper if they want accurate stats for the index advisor.